### DISCUSSION TITLE
Dictionary transforms for images of different resolution

### DISCUSSION THREAD

**Discussion by RohanKPatil** [Project-MONAI/MONAI#1867] [Ideas]
Is there support available for performing affine transforms for dictionary based methods for images with different resolutions? This would be helpful for image restoration problems

**Comment by Nic-Ma** [comment]
Hi @RohanKPatil ,

Thanks for your interest here.
Currently, for spatial transforms, only `spacingd` and `orientationd` dict transforms can update affine.

Thanks.
